udata/templates/territories/territory.html

```diff
@@ -46,5 +46,5 @@
 {% block extra_js %}
     <script src="{{ static('vue-common.js') }}"></script>
     <script src="{{ static('territory.js') }}"></script>
-    <script src="{{ static('widgets.js', _burst=False) }}" id="udata" async defer onload="udataScript.load()"></script>
+    <script src="{{ static('widgets.js', _burst=False) }}" id="udata" async defer onload="udataScript.loadDatasets()"></script>
```

Comment:
Does the embed code (the one proposed to the user with the button) need the same modification ?